elfloader-tool/src/arch-arm/smp_boot.c

```diff
@@ -134,6 +144,17 @@ void smp_boot(void)
     arm_disable_dcaches();
 #endif
     init_cpus();
+
+#if defined(CONFIG_ARCH_AARCH64)
+    dsb();
+#endif
+
     non_boot_lock = 1;
+
+#if defined(CONFIG_ARCH_AARCH64)
+    /* Secondary CPUs may still run with MMU & caches off. Force the update to be visible. */
```

Comment:
The comment applies in general, not just for CONFIG_ARCH_AARCH64. Seems just for AARCH64 we have the proper command here, for ARMv7/AARCH there is still a open ToDo to look up the proper command/function one day can call it here.

Reply:
Sure, I'll move the comment out